libs/domain-service/src/__tests__/unit/resolvers.unit.test.ts

```diff
@@ -1,33 +1,45 @@
-import network from "@ledgerhq/live-common/network";
```

Comment:
Why do we now use axios instead of network ?

Reply:
Same reason explained here: https://github.com/LedgerHQ/ledger-live/pull/3059#discussion_r1149021802